tests/async/tasync_traceback.nim

```diff
@@ -110,13 +110,15 @@ Async traceback:
       ## Executes pending callbacks
     asyncmacro\.nim\(\d+?\)\s+?fooNimAsyncContinue
       ## Resumes an async procedure
-    tasync_traceback\.nim\(\d+?\)\s+?fooIter
+    asyncmacro\.nim\(\d+?\)\s+?fooIter
```

Comment:
This actually reduces the amount of signal we get from the async tracebacks. I cannot recall whether this particular line provided useful information, but if it does then that could be a blocker for these changes. Can you run this test and show us what the tracebacks are before and after your changes?

Reply:
IIRC the change is similar to what is seen in the diff, basically we need to add the linedir of `tasync_traceback` to the result of quote

Reply:
I want to see what line the traceback points to now.

Reply:
`asyncmacro.nim(261)  `

Reply:
which seems to be `(cast[type(f)](internalTmpFuture)).read()`

Reply:
That should be okay, I'm assuming that `tasync_traceback` in the old traceback doesn't point to anything interesting though.